public/js/index.js

```diff
@@ -106,36 +105,198 @@ const cursorActivityDebounce = 50
 const cursorAnimatePeriod = 100
 const supportContainers = ['success', 'info', 'warning', 'danger']
 const supportCodeModes = [
```

Comment:
We could drop the hardcoding here: https://github.com/hedgedoc/hedgedoc/pull/21/files#diff-93842c4bc014d741ce85adbcd0da539017de00a76b65956e403c4732d925788fR95

Reply:
I'm not sure if I understand.
I explicitly hardcoded the list of languages to get rid of the import of `highlight.js`, as it can't be lazy-loaded and otherwise would get included in all bundles that contain `index.js`,